### ISSUE TITLE
Using rustc --print sysroot

### ISSUE THREAD

**Issue by bkchr** [mozilla/nixpkgs-mozilla#16]
First thanks for the awesome overlay! 
I installed rust and rust-src from the stable channel. 
Now I wanted to use "rustc --print sysroot", but both packages are installed into different directories, so that is not working. Do you have any ideas how I could accomplish this? Or is there a way to export the rust-src path as env variable?

**Comment by zimbatm** [comment]
> Now I wanted to use "rustc --print sysroot", but both packages are installed into different directories, so that is not working

It's not clear to me, what is the use-case?

**Comment by bkchr** [comment]
Some programs need to get the path to the rust-src and use rustc --print sysroot to construct this path.

**Comment by bkchr** [comment]
As far as I understand, installing the package rustChannels.stable.rust should install everything in on package? But if I install this package and try to search for the rust-src, I can not find anything :(

**Comment by nbp** [comment]
This is now fixed by #18 by overriding the `extensions` argument of the derivation with the list of extensions that you want to install, as follow:

```nix
self: super:

{
  myRust = super.rustChannels.nightly.rust.override {
    extensions = [ "rust-src" ];
  };
}
```
